vpn-network/vpn-network-impl/src/main/java/com/duckduckgo/vpn/network/impl/models/ResourceRecord.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.duckduckgo.vpn.network.impl.models
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+class ResourceRecord {
```

Comment:
This is class used by native code to pass information.
I have not make it a eg. `data class` to not mess with the generated bytecode.